README.adoc

```diff
@@ -461,6 +471,65 @@ operator-sdk build $IMAGE --enable-tests && docker push $IMAGE
 operator-sdk test cluster $IMAGE --namespace $NAMESPACE --service-account unifiedpush-operator
 ....
 
+== Publishing images
+
+Images are automatically built and pushed to our https://quay.io/repository/aerogear/unifiedpush-operator[image repository] by the Jenkins in the following cases:
+
+- For every change merged to master a new image with the `master` tag is published.
+- For every change merged that has a git tag a new image with the `<operator-version>` and `latest` tags are published.
+
+== Tags Release
+
+Following the steps
+
+. Create a new version tag following the http://semver.org/spec/v2.0.0.html[semver], for example `0.1.0`
+. Bump the version in the link:./version/version.go[version.go] file.
+. Update the the link:./CHANGELOG.MD[CHANGELOG.MD] with the new release.
+. Looking for the SOPs and update the tag for the them in all files (e.g `https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc`)
+. Create a git tag with the version value, for example:
++
+[source,shell]
+----
+$ git tag -a 0.1.0 -m "version 0.1.0"
+----
++
+. Push the new tag to the upstream repository, this will trigger an automated release by the Jenkins, for example:
++
+[source,shell]
+----
+$ git push upstream 0.1.0
+----
++
+NOTE: The image with the tag will be created and pushed to the https://quay.io/repository/aerogear/unifiedpush-operator[unifiedpush-operator image hosting repository] by the Jenkins.
+++
+WARNING: Do not use letters in the tag such as `v`. It will not work.
```

Comment:
Why would it not work? I'd have expected any tags to work :thinking:

_If_ it's true that it wouldn't work, then I think this warning would be better placed immediately above where the instructions to create the tag are described, rather than 15 lines later.

Reply:
It does not work in the previous impl. I will remove from here because I do not know in Jenkins.